src/composables/useUtilities.ts

```diff
@@ -160,12 +165,15 @@ export default function useUtilities() {
     const state = ref(swrvState.PENDING)
 
     watchEffect(() => {
-      const hasData =
-        response.value?.data?.length ||
-        response.value?.data?.data?.length ||
-        (!response.value?.data?.data &&
-          typeof response.value?.data === 'object' &&
+      // TODO: Determine the proper "generic" way to determine if data exists
```

Comment:
I originally added this TODO; maybe we can figure this one out together next week

Reply:
@kaiarrowood we can probably remove this TODO since we're merging this PR